Three missing modules need to be added to setup.py to be installed.
Patch was developed in-house; it is Solaris specific and
will not be contributed upstream.

--- python-ldap-2.4.25/setup.py.old	2016-07-19 10:13:07.016090954 -0700
+++ python-ldap-2.4.25/setup.py	2016-07-19 10:13:49.738738316 -0700
@@ -153,6 +153,7 @@
     'ldap.controls.deref',
     'ldap.controls.libldap',
     'ldap.controls.openldap',
+    'ldap.controls.pagedresults',
     'ldap.controls.ppolicy',
     'ldap.controls.psearch',
     'ldap.controls.pwdpolicy',
@@ -159,6 +160,8 @@
     'ldap.controls.readentry',
     'ldap.controls.sessiontrack',
     'ldap.controls.simple',
+    'ldap.controls.sss',
+    'ldap.controls.vlv',
     'ldap.cidict',
     'ldap.dn',
     'ldap.extop',
